apps/schema/src/components/ProfileManager.vue

```diff
@@ -0,0 +1,343 @@
+                    }}</i></small
+                  >
+                </div>
+              </li>
+            </template>
+          </ul>
+        </div>
+      </div>
+      <div class="col-2">
+        <h2>My selection:</h2>
+        <a href="" @click.prevent="downloadCsv()">download</a>
+        <ul>
+          <li v-for="table in selectedTables">
+            {{ table.name }}
+            <ul v-if="table.columns?.some((c) => c.selected)">
+              <li v-for="column in table.columns?.filter((c) => c.selected)">
+                {{ column.name }}
+              </li>
+            </ul>
+          </li>
+        </ul>
+      </div>
+    </div>
+  </div>
+</template>
+
+<style scoped>
+.list-group li:hover {
+  cursor: pointer;
+}
+</style>
+
+<script setup>
+import { ref, computed } from "vue";
+import { InputSearch, MessageError, ButtonAlt } from "molgenis-components";
+import VueScrollTo from "vue-scrollto";
+
+const error = ref(null);
+const profileJson = ref({});
+const selectedProfiles = ref([]);
+const search = defineModel();
+
+fetch("../api/profiles")
+  .then((response) => response.json())
+  .then((data) => {
+    profileJson.value = data;
+    profileJson.value.tables.forEach(
+      (t) => (t.columns = t.columns?.filter((c) => !c.inherited))
+    );
+  })
+  .catch((err) => (error.value = err));
+
+const tables = computed(() => {
+  if (profileJson.value) {
+    return profileJson.value.tables?.filter(
+      (t) =>
+        ((!search.value || JSON.stringify(t).includes(search.value)) &&
+          selectedProfiles.value.length == 0) ||
+        selectedProfiles.value.some(
+          (p) =>
+            t.profiles?.includes(p) ||
+            t.columns?.some((c) => c.profiles?.includes(p))
+        )
+    );
+  }
+});
+
+const selectedTables = computed(() => {
+  if (profileJson.value) {
+    return profileJson.value.tables?.filter(
+      (t) => t.selected || t.columns?.some((c) => c.selected)
+    );
+  }
+});
+
+const profiles = computed(() => {
+  const result = [];
+  if (!profileJson.value.tables) {
+    return [];
+  }
+  profileJson.value.tables.forEach((t) => {
+    if (t.profiles) {
+      result.push(...t.profiles);
+    }
+    if (t.columns) {
+      t.columns.forEach((c) => {
+        if (c.profiles) {
+          result.push(...c.profiles);
+        }
+      });
+    }
+  });
+  return Array.from(new Set(result));
+});
+
+const isVisible = function (column) {
+  return (
+    (selectedProfiles?.value.length == 0 ||
+      column.profiles?.some((p) => selectedProfiles?.value.includes(p))) &&
+    (!search.value || JSON.stringify(column).includes(search.value))
+  );
+};
+
+const v = function (value) {
+  if (value == null) {
+    return "";
+  } else if (Array.isArray(value)) {
+    if (value.length > 1) {
+      return `"${value.join(",")}"`;
+    } else {
+      return v(value[0]);
+    }
+  } else {
+    return value;
+  }
+};
+
+const selectAll = function (tableName) {
+  tables.value.forEach((t) => {
```

Comment:
try not to use single letter variables, `table` is not that verbose that it warrants shortening :)

Reply:
I think you guys are really overdoing the review now, is not worth it ;-) My question for review was to make sure that I didn't break anything. This is a poc.

Reply:
Btw on topic: i like short iterator names like 'i' and 'c' and 't' so I know I am dealing with an iterator. But if majority feels like this I don't mind to change the guidebook on this.